serverless/integrations/gpu-resources.adoc

```diff
@@ -0,0 +1,39 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="gpu-resources"]
+= Using NVIDIA GPU resources with serverless applications
+:context: gpu-resources
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+NVIDIA support experimental use of GPU resources on {product-title}.
+See link:https://docs.nvidia.com/datacenter/kubernetes/openshift-on-gpu-install-guide/index.html[{product-title} on NVIDIA GPU accelerated clusters] for more information about setting up GPU resources on {product-title}.
+
+Once GPU resources are enabled for your {product-title} cluster, you can specify GPU requirements for a Knative service using the `kn` CLI.
+
+.Specifying GPU requirements for a Knative service using `kn`
+
+You can specify a GPU resource requirement when you create a Knative service using `kn`.
```

Comment:
I see that you're writing out the task, rather than using `Procedure` as the subhead on this page. As written, the first sentence repeats the header, so one alternative is to delete it. But I'd suggest changing the header to `Procedure` instead to mirror our other docs.

Reply:
100%, great catch, especially since we already say what the procedure will do. Thank you!